src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetObjectForNativeVariantTests.Windows.cs

```diff
@@ -538,7 +538,7 @@ public void GetObjectForNativeVariant_NestedVariant_ReturnsExpected(Variant sour
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/27015", TargetFrameworkMonikers.Netcoreapp)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp, "VT_RECORD not supported on .NET Core")]
```

Comment:
These tests run on .NET Core only. If they are not relevant for .NET Core, delete them (or change them to check for behavior expected on .NET Core).

Reply:
This code path cannot be run on .NET Core due to the reasons mentioned in https://github.com/dotnet/runtime/issues/27015#issuecomment-611079237. Will remove this specific test.

Reply:
Instead I changed the test to validate the failure.